lazyflow/utility/testing.py

```diff
@@ -139,3 +140,76 @@ def propagateDirty(self, slot, subindex, roi):
             raise
         finally:
             self.Output.setDirty(roi)
+
+
+@dataclass
+class SlotDescription:
+    dtype: np.dtype = None
+    shape: Tuple[int] = (1,)
+    axistags: vigra.AxisTags = None
+    level: int = 0
+    data: Any = None
+
+
+SLOT_DATA = Mapping[str, SlotDescription]
```

Comment:
Uppercased naming is somewhat misleading because `Mapping[..., ...]` is not "generic".

```suggestion
SlotData = Mapping[str, SlotDescription]
```